skbeam/core/roi.py

```diff
@@ -511,3 +518,76 @@ def extract_label_indices(labels):
     label_mask = labels[labels > 0]
 
     return label_mask, pixel_list
+
+
+def bar_rois(edges, values):
+    """Draw bar shaped roi's when the edges are provided
+
+    Parameters
+    ----------
+    edges : list
+        giving the inner and outer edges of each bar
+        e.g., [(1, 2), (11, 12), (21, 22)]
+    values : array
+        image pixels co-ordinates to create bar rois.
+        This bars can be horizontal or vertical depend on the edges
+
+    Returns
+    -------
+    label_array : array
+        Elements not inside any ROI are zero; elements inside each
+        ROI are 1, 2, 3, corresponding to the order they are
+        specified in edges.
+
+    Note
+    ----
+    These roi's can be created using the rings function
+    """
+    return rings(edges, values, values.shape)
+
+
+def box_rois(v_values, v_edges, h_values=None, h_edges=None):
```

Comment:
I would change this API to be:

```def box(v_edges, h_edges, shape)```

The set up for `v_values` and `h_values` is not intuitive and we can do that inside this function